onnx/backend/test/runner/__init__.py

```diff
@@ -174,9 +174,9 @@ def _assert_similar_outputs(ref_outputs, outputs, rtol, atol):  # type: (Sequenc
                 rtol=rtol,
                 atol=atol)
 
-    @staticmethod
+    @classmethod
     @retry_excute(3)
-    def _download_model(model_test, model_dir, models_dir):  # type: (TestCase, Text, Text) -> None
+    def _download_model(cls, model_test, model_dir, models_dir):  # type: (TestCase, Text, Text) -> None
```

Comment:
same here, if we turn it to classmethod, that means it can be overridden. Let's turn it public